services/gitea/gitea-base.js

```diff
@@ -0,0 +1,32 @@
+import { BaseJsonService } from '../index.js'
+
+export default class GiteaBase extends BaseJsonService {
+  static auth = {
+    passKey: 'gitea_token',
+    serviceKey: 'gitea',
+  }
+
+  async fetch({ url, options, schema, httpErrors }) {
+    return this._requestJson(
+      this.authHelper.withBearerAuthHeader({
+        schema,
+        url,
+        options,
+        httpErrors,
+      }),
+    )
+  }
+
+  async fetchPage({ page, requestParams, schema }) {
```

Comment:
This method does not appear to be used anywhere

Reply:
Good catch, I missed adding pagination to the API.